src/components/Balance.tsx

```diff
@@ -19,19 +20,42 @@ const Balance: React.FC<BalanceProps> = ({
   unit,
   prefix,
   onClick,
+  fixedWidth,
   ...props
 }) => {
   const previousValue = useRef(0)
+  const containerStyles = () => {
+    if (fixedWidth) {
+      return {
+        fontVariantNumeric: 'tabular-nums',
+        fontFamily: 'sans-serif',
```

Comment:
Need to run this by Chef Bugs.

Reply:
It's a no from me, as the mix of fonts is visually jarring and there are other potential workarounds for this specific case.
Long term, I'm looking at other fonts that cover all our languages and include monospace numerals. Will return to that discussion at a later date.

Reply:
https://github.com/pancakeswap/pancake-frontend/pull/1718

You can review this